README.md

```diff
@@ -12,14 +12,28 @@ catkin_make
 ```
 
 ## Running
-If you have a .world file, place it in `/worlds` and modify the `ttb3_velodyne.launch` file to load your world.
+If you have a .world file, place it in `/worlds` and modify the respective `.launch` file to load your world.
 
 ```xml
 <arg name="world_name" value="$(find turtlebot3_ramp_sim)/worlds/your_world_here.world"/>
 ```
+
 Launch the simulation
+
 ```ros
-roslaunch turtlebot3_ramp_sim ttb3_velodyne.launch
+roslaunch turtlebot3_ramp_sim ttb3_(sensor name).launch
 ```
+
 You should see your world in Gazebo and the visualization of the LIDAR in RVIZ.
 
+## Specific instructions for RealSense attachment
+In order to run the simulation with RealSense camera, the Gazebo plugin for D435 is required.
+
+```
+cd ~/catkin_ws/src
+git clone https://github.com/pal-robotics/realsense_gazebo_plugin.git
```

Comment:
Is there a binary package for this which can be directly installed using an apt install command?

Reply:
Don't think there's a binary package for the Gazebo plugin, only for the physical sensors.